src/components/ProductCard.vue

```diff
@@ -46,7 +46,7 @@
 
     <div
       v-for="(theInfo, index) in consumerInfo"
```

Comment:
```suggestion
      v-for="theInfo in consumerInfo"
```
@coopjakob like this ^ :fire:

Reply:
```
    ```suggestion
          v-for="theInfo in consumerInfo"
    ```
```